test/modules/CloudToDeviceMessageTester/Settings.cs

```diff
@@ -54,7 +54,8 @@ class Settings
                     ApiVersion = Preconditions.CheckNonWhiteSpace(apiVersion, nameof(apiVersion)),
                     ModuleGenerationId = Preconditions.CheckNonWhiteSpace(moduleGenerationId, nameof(moduleGenerationId)),
                     IotHubHostName = Preconditions.CheckNonWhiteSpace(iotHubHostName, nameof(iotHubHostName)),
-                    TransportType = transportType
+                    TransportType = transportType,
+                    EdgeDeviceId = deviceId
```

Comment:
nit: both should be edgeDeviceId?

Reply:
deviceId is read from env variable and for leaf device is using that name and appends "-leaf", so deviceId is actually edgeDeviceId